<launch>
  <node name="slamwareNode"          pkg="slamwareAPP"  type="slamwareNode" output="screen" respawn="true">

  <param name = "ip_addres"                value = "192.168.11.1"/>
  <param name = "angle_compensate"         value = "false"/>
  <param name = "fixed_odom_map_tf"         value = "false"/>

  <param name = "robot_frame"              value = "/base_link"/>
  <param name = "odom_frame"               value = "/odom"/>
  <param name = "laser_frame"              value = "/laser"/>
  <param name = "map_frame"                value = "/slamware_map"/>

  <param name = "robot_pose_pub_period"    value = "0.05"/>
  <param name = "scan_pub_period"          value = "0.1"/>
  <param name = "map_pub_period"           value = "0.2"/>
  <param name = "path_pub_period"          value = "0.05"/>

  <param name = "map_size_down_left_x"    value = "-10"/>
  <param name = "map_size_down_left_y"    value = "-10"/>
  <param name = "map_size_width"          value = "25"/>
  <param name = "map_size_height"         value = "25"/>

  <param name = "/cmd_vel"                 value = "/cmd_vel"/>
  <param name = "/move_base_simple/goal"   value = "/move_base_simple/goal"/>
  <param name = "scan"                     value = "scan"/>
  <param name = "odom"                     value = "odom"/>
  <param name = "map"                      value = "map"/>
  <param name = "map_metadata"             value = "map_metadata"/>
  <param name = "global_plan_path"         value = "global_plan_path"/>

  <!-- topic remap /-->
  <remap from = "/cmd_vel"                   to = "/cmd_vel"/>
  <remap from = "/move_base_simple/goal"     to = "/move_base_simple/goal"/>
  <remap from = "scan"                       to = "scan"/>
  <remap from = "odom"                       to = "odom"/>
              =                                 =
  <remap from = "map"                        to = "slamware_map"/>
  <remap from = "map_metadata"               to = "map_metadata"/>
  <remap from = "global_plan_path"           to = "global_plan_path"/>
  <remap from = "slamwareNode/guide"         to = "guide"/>
  <remap from = "slamwareNode/tts"           to = "tts"/>
  <remap from = "slamwareNode/battery"       to = "battery"/>

  </node>
<!--
  <node name="map2odom" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 /slamware_map /odom 100"/>
  <node name="base2laser" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 /base_link /laser 100"/>-->



</launch>
